lib/iex/lib/iex/info.ex

```diff
@@ -43,6 +43,22 @@ defimpl IEx.Info, for: Atom do
     [{"Data type", "Atom"}] ++ description ++ specific_info
   end
 
+  defp module?(atom) do
+    case :code.get_object_code(atom) do
+      :error ->
+        Code.ensure_loaded?(atom)
+
+      {^atom, beam, _path} ->
+        module_name =
```

Comment:
What do you think about using `Keyword.fetch` to avoid the comment and the default `0` value?

```elixir
beam_info = :beam_lib.info(beam)

case Keyword.fetch(beam_info, :module) do
  {:ok, ^atom} -> true
  _ -> false
end
```

Reply:
Changed to `match?/2`.